README.md

```diff
@@ -23,8 +23,8 @@ The ```master``` branch of this repository contains samples configured for the l
 ## Requirements
 
 * [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) 100.4.0 (or higher).
-* Xcode 10.0 (or higher)
-* iOS 12 SDK (or higher)
+* Xcode 10.1 (or higher)
+* iOS 12.1 SDK (or higher)
```

Comment:
This isn't related to this PR, but we should consider removing the minimum version of the iOS SDK needed to build the app and replacing it with the minimum version of iOS required to run the app. The min Xcode version implies the SDK version anyway.